package.json

```diff
@@ -35,61 +40,62 @@
     "build": "webpack --mode production",
     "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
     "lint": "eslint .",
-    "update": "npx npm-check -u"
+    "update": "npx npm-check -u",
+    "prepare": "husky install"
   },
   "browserslist": [
     "last 2 versions"
   ],
+  "engines": {
+    "node": ">=12.20.0"
+  },
   "dependencies": {
-    "angular": "npm:@xlts.dev/angular@1.8.2",
+    "angular": "npm:@xlts.dev/angular@1.8.7",
     "core-js": "^3.13.0"
   },
   "devDependencies": {
-    "@babel/core": "7.10.2",
-    "@babel/plugin-proposal-object-rest-spread": "7.10.1",
-    "@babel/plugin-transform-object-assign": "7.10.1",
-    "@babel/preset-env": "7.10.2",
-    "angular-mocks": "npm:@xlts.dev/angular-mocks@1.8.2",
-    "autoprefixer": "^9.8.6",
+    "@babel/core": "7.16.7",
+    "@babel/plugin-proposal-object-rest-spread": "7.16.7",
+    "@babel/plugin-transform-object-assign": "7.16.7",
+    "@babel/preset-env": "7.16.8",
+    "angular-mocks": "npm:@xlts.dev/angular-mocks@1.8.7",
+    "autoprefixer": "10.4.2",
     "babel-core": "^6.26.3",
     "babel-eslint": "^10.1.0",
     "babel-loader": "^8.2.2",
     "babel-plugin-angularjs-annotate": "^0.10.0",
-    "babel-plugin-ng-hot-reload": "^2.1.0",
-    "clean-webpack-plugin": "^3.0.0",
-    "css-loader": "^5.2.6",
-    "eslint": "^7.27.0",
+    "clean-webpack-plugin": "4.0.0",
+    "css-loader": "6.5.1",
+    "eslint": "8.6.0",
     "file-loader": "^6.2.0",
-    "html-loader": "^1.1.0",
-    "html-webpack-plugin": "^4.3.0",
-    "husky": "^4.3.8",
+    "html-loader": "3.1.0",
+    "html-webpack-plugin": "5.5.0",
+    "husky": "7.0.4",
     "imagemin": "^8.0.0",
     "img-loader": "^4.0.0",
-    "jasmine-core": "^3.7.1",
+    "jasmine-core": "4.0.0",
     "karma": "^6.3.2",
     "karma-chrome-launcher": "^3.1.0",
     "karma-jasmine": "^4.0.1",
-    "karma-webpack": "^4.0.2",
-    "lint-staged": "^11.0.0",
-    "mini-css-extract-plugin": "^0.12.0",
+    "karma-webpack": "5.0.0",
+    "lint-staged": "12.1.7",
+    "mini-css-extract-plugin": "2.4.7",
+    "ng-hot-reload-loader": "^3.3.1",
     "postcss": "^8.3.0",
-    "postcss-loader": "^4.3.0",
+    "postcss-loader": "6.2.1",
     "prettier": "^2.3.0",
     "sass": "^1.35.1",
-    "sass-loader": "^8.0.2",
+    "sass-loader": "12.4.0",
     "url-loader": "^4.1.1",
-    "webpack": "^4.46.0",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.2"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged && npm run lint"
-    }
+    "webpack": "5.66.0",
+    "webpack-cli": "4.9.1",
+    "webpack-dev-server": "4.7.3"
   },
   "lint-staged": {
     "*.{js,json,css,scss,less,md,ts,html,component.html}": [
       "prettier --write"
-    ]
+    ],
+    "*.js": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
```

Comment:
This is repeated because `eslint --fix` might change things that need to be re-prettier'd?

Reply:
I didn't notice this. I've updated this configuration now that you've pointed it out. This was added automatically by running:
```bash
npx mrm@2 lint-staged
```
Which is the "_The fastest way to start using lint-staged_" per their instructions: https://github.com/okonet/lint-staged#installation-and-setup